### PR TITLE
adds robot command verb to select a voice pitch

### PR DESCRIPTION
**Pull request by scrcr0**
## About the PR 
Rethink of a previous PR. Borgs can now use a robot command verb to change the pitch of their "voice" from three options. 

[Demo](https://www.youtube.com/watch?v=BipLLJ_Vp84)

image

## Why's this needed? 
With borgs allowed to express themselves through their names, clothing and facial expression (when equipped with the screen head), I thought it would be appropriate for them to choose a vocal pitch that better fits with their self expression. 

## Changelog

```changelog
(u)zcrow
(*)Added "Change vocal pitch" verb to Robot Commands. Borgs can now modify their voices to better express themselves. 
```

### CODE REVIEW

**Comment by scrcr0** [comment]
okay... classic me vs git issue. need to fix that wonky commit

**Comment by frawhst** [comment]
wait robots use `get_age_pitch()`???? do we even set their age anywhere
if not feel free to remove that and make this the only source of robot pitch modification
feature is good in general though

**Comment by scrcr0** [comment]
> wait robots use `get_age_pitch()`???? do we even set their age anywhere if not feel free to remove that and make this the only source of robot pitch modification feature is good in general though

apparently so. removing it had no effect, so removed!

File: code/mob/living/silicon/robot.dm [hunk 1]
```diff
@@ -2217,6 +2219,20 @@
 		else
 			boutput(usr, "<span class='alert'><b>Internal PDA not found!</span>")
 
+	verb/change_voice_pitch()
+		set category = "Robot Commands"
+		set name = "Change vocal pitch"
+
+		var/list/vocal_pitches = list("Low", "Medium", "High")
+		var/vocal_pitch_choice = input("Select a vocal pitch", "Robot", null, null) in vocal_pitches
```
Comment (flrsh) [inline code/mob/living/silicon/robot.dm:2227]: Could you use `tgui_input_list`?

Comment (scrcr0) [reply code/mob/living/silicon/robot.dm]: Absolutely. Updated in latest commit

**Review by flrsh — APPROVED** [review]

**Pull request merged**